cli/pcluster/config/pcluster_config.py

```diff
@@ -111,7 +111,10 @@ def _init_config_parser(self, config_file, fail_on_config_file_absence=True):
         else:
             LOGGER.debug("Parsing configuration file %s", self.config_file)
         self.config_parser = configparser.ConfigParser(inline_comment_prefixes=("#", ";"))
-        self.config_parser.read(self.config_file)
+        try:
+            self.config_parser.read(self.config_file)
+        except configparser.ParsingError as e:
+            LOGGER.debug("Error parsing configuration file {0}.\n{1}".format(self.config_file, e.message))
```

Comment:
Should this be a warning instead of a debug? I could be mistaken, but I don't think that the debug messages are shown by default.

Reply:
Yes, errors managing and display for pcluster configure has been deeply revisited now (and further changes will be probably needed in the future). By now we are focusing on displaying only informative messages to the user, avoiding low level ones like parsing errors.